delatore/sources/base.py

```diff
@@ -22,19 +22,29 @@ class NoUpdates(Exception):
     """Raised when source has no updates"""
 
 
+class Topics(NamedTuple):
+    changes: str
+    info: str
+
+    @classmethod
+    def get_prefix(cls, prefix: str):
```

Comment:
`get_prefix` is when you return prefix, in this case it is `from_prefix`, `with_prefix`, etc.